Sources/DatadogCrashReporting/PLCrashReporterIntegration/CrashReport.swift

```diff
@@ -232,14 +232,12 @@ extension ThreadInfo {
 
 extension BinaryImageInfo {
     init?(from imageInfo: PLCrashReportBinaryImageInfo) {
-        guard let imagePath = imageInfo.imageName,
-              let imageUUID = imageInfo.hasImageUUID ? imageInfo.imageUUID : nil else {
-            // We can drop this image as it won't be useful for symbolication - both
-            // "image name" and "uuid" are necessary.
+        guard let imagePath = imageInfo.imageName else {
+            // We can drop this image as it won't be useful for symbolication
             return nil
         }
 
-        self.uuid = imageUUID
+        self.uuid = imageInfo.imageUUID
```

Comment:
Without `NS_ASSUME_NONNULL_*` region or `_Nonnull` objc annotation, the property is [implicitly unwrapped optional](https://developer.apple.com/documentation/swift/objective-c_and_c_code_customization/designating_nullability_in_objective-c_apis), so this won't crash as `uuid` is now optional. The previous check was not necessary since the `guard let` was already evaluating nullability.
But I can checks in test tho 👍

Reply:
It's actually already covered in test :)
https://github.com/DataDog/dd-sdk-ios/blob/3b43f9141d2415224fd348e6d10948b245c41853/Tests/DatadogCrashReportingTests/Mocks.swift#L144